### PR TITLE
Iss37 fix place report

### PR DESCRIPTION
**Pull request by LPMendoza**
# Description
A filter to the places report for the api has been added. Now the report only shows  the places wich are from the same institution of the user logued.
 
image

### CODE REVIEW

**Review by MiguelOcegueraM — APPROVED** [review]

File: Gemfile [hunk 1]
```diff
@@ -64,5 +63,9 @@ group :test do
   gem 'webdrivers'
 end
 
+group :test, :development do
+  gem 'rspec-rails', '~> 5.0.0'
```
Comment (OchoaHub) [inline Gemfile:67]: Rspec > Minitest

**Review by OchoaHub — APPROVED** [review]
Looks fine!

**Review by LDVera — APPROVED** [review]

**Review by jnegrete15 — APPROVED** [review]

**Review by fego-dev — APPROVED** [review]
Nice, good job.

**Review by LDVera — APPROVED** [review]

File: app/controllers/api/reports_controller.rb [hunk 2]
```diff
@@ -10,7 +10,9 @@ def places_report
                 time_to = params[:time_to]
                     time_from = params[:time_from]
             end
+            user = current_api_user
             @incidents = Incident.select("place_id, SUM(symptomatic) AS symptomatic, SUM(covid_positive) AS positive , SUM(covid_negative) AS negative").where("created_at between '#{time_from}'::timestamp and '#{time_to}'::timestamp").order('symptomatic DESC, negative DESC, positive DESC').group(:place_id);
+            @incidents = @incidents.filter { |inci| inci.place.institute.id == user.institute_id}
```
Comment (ArielMD) [inline app/controllers/api/reports_controller.rb:15]: Nice job, good, you only show the incidents of the current user

**Review by ArielMD — APPROVED** [review]
Nice job

**Pull request merged**